pom.xml

```diff
@@ -173,7 +173,7 @@
     <dependency>
       <groupId>org.jenkins-ci.plugins</groupId>
       <artifactId>scm-api</artifactId>
-      <version>1.0</version>
+      <version>1.3-SNAPSHOT</version> <!-- TODO https://github.com/jenkinsci/scm-api-plugin/pull/12 -->
```

Comment:
Yes, but @stephenc is on vacation for another week so I am considering pushing for a https://github.com/jenkinsci/scm-api-plugin/pull/12 release before then.